test/artefacts/pay-leave-for-parents/yes/2018-02-01/worker/worker/yes/yes/10000.0-year/yes/yes/yes/10000.0-year/yes.txt

```diff
@@ -30,10 +30,6 @@ The mother and her partner can both get [shared parental pay](/shared-parental-l
 
 ###Amount
 
-Mother’s shared parental pay (between 6 April 2015 and 5 April 2016) | £139.58 per week or 90% of their average weekly earnings (before tax), whichever is lower
-
```

Comment:
@ikennaokpala It's because the original logic says:
if the due date is in not 2013-2014, or not 2014-2015 or not 2015-2016, then display this text. As 2018 doesn't fall in these ranges, the text is displayed.
In the new logic, the text is only displayed if due date IS in 2016-2017, so the text is removed for 2018.